be/src/common/config.h

```diff
@@ -107,6 +107,9 @@ CONF_String(mem_limit, "90%");
 // Enable the jemalloc tracker, which is responsible for reserving memory
 CONF_Bool(enable_jemalloc_memory_tracker, "true");
 
+// Enable the RSS non jemalloc tracker, which tracks process RSS - jemalloc.resident
+CONF_Bool(enable_rss_non_jemalloc_tracker, "true");
```

Comment:
The default should be false. Otherwise, it may cause existing queries to fail.

Reply:
Why ?

Reply:
This portion of memory may be fragmented, in which case allocating content does not necessarily require creating a new page.